RELEASE_NOTES.md

```diff
@@ -1,5 +1,72 @@
 <!-- markdown-link-check-disable -->
 
+# 0.75.0
+
+The `0.75.0` release introduces dashboard enhancements for stack components 
```

Comment:
```suggestion
The `0.75.0` release introduces dashboard enhancements for stack component
```

Reply:
I will commit this. But, I will wait for the CI to finish.